website/cue/reference/remap/literals/string.cue

```diff
@@ -48,8 +52,19 @@ remap: literals: string: {
 				"\\0":       "Null"
 				"\\\"":      "Double quote"
 				"\\'":       "Single quote"
+				"\\{{":      "Double brace"
 			}
 		}
+		templates: {
+			title: "Templates"
+			description: """
+				Strings can be templated by enclosing a variable name with `{{..}}`. The
+				value of the variable is inserted into the string at this position at runtime.
+				The variable has to be a string. Only variables are supported, if you want to
+				insert a path from the event you must assign it to a variable first. To insert
+				a `{{` into the string it can be escaped with a `\\` escape: `\\{{..\\}}`.
+				"""
```

Comment:
I might leave a note that we plan to expand this in the future to allow paths and format strings.